pages/en/advanced/mina-signer.mdx

```diff
@@ -20,35 +20,37 @@ Never reuse the same keypair from a testnet on mainnet!
 
 ## Installation
 
-The source code for our client SDK is open source and be seen [here](https://github.com/MinaProtocol/mina/tree/develop/frontend/client_sdk).
+The source code for our SDK is open source and be seen [here](https://github.com/MinaProtocol/mina/tree/develop/frontend/mina-signer).
 
-To get the client SDK installed, you will need NodeJS on your system. Make sure it is available on your machine and then use either yarn or npm to install the client SDK.
+To get the SDK installed, you will need NodeJS on your system. Make sure it is available on your machine and then use either yarn or npm to install the Mina Signer.
 
 ```bash
 # yarn:
-yarn add @o1labs/client-sdk
+yarn add mina-signer
 
 # npm:
-npm install --save @o1labs/client-sdk
+npm install --save mina-signer
 ```
 
 ## Usage
 
-Creating a key pair is simple with our client SDK. See the snippet below to see how to get create a key pair.
+Creating a key pair is simple with the Mina Signer. See the snippet below to see how to get create a key pair.
 
 ### NodeJS:
```

Comment:
Might suggest `Create a key pair` as a heading b/c NodeJS is confusing unless we plan to show it for non-NodeJS platforms. It also creates a pattern that works if we wanted to add additional sections showing how to use other functionality